src/realizations/catchment/Bmi_C_Adapter.cpp

```diff
@@ -402,6 +402,54 @@ std::string Bmi_C_Adapter::GetVarUnits(std::string name) {
     return units_str;
 }
 
+std::string Bmi_C_Adapter::GetVarLocation(std::string name) {
+    char location_c_str[BMI_MAX_UNITS_NAME];
```

Comment:
Again, might want to consider a `BMI_MAX_LOCATION_SIZE`

Reply:
ok